example/script/generate-etch-sign-url.js

```diff
@@ -0,0 +1,37 @@
+const Anvil = require('../../src/index')
+const argv = require('yargs')
+  .usage('Usage: $0 apiKey signerEid clientUserId')
+  .demandCommand(3).argv
+
+const [apiKey, signerEid, clientUserId] = argv._
+
+async function main () {
+  const clientOptions = {
+    apiKey,
+  }
+
+  const client = new Anvil(clientOptions)
+
+  const variables = {
+    signerEid,
+    clientUserId,
+  }
+
+  const { statusCode, data, errors } = await client.generateEtchSignUrl({ variables })
```

Comment:
Very minor thing, but seems a little funny to me to have the URL returned be in a thing called `data`...perhaps the `generateEtchSignUrl` method could/should massage the response to rename it to `url`?  Wonder what @benogle thinks...

E.g.
```js
async generateEtchSignUrl(...) {
  const { data, ...rest } = await this.requestGraphQL(...)
  return {
    url: data,
    ...rest
  }
}
```

Reply:
I think this would be better